pkg/autoscaler/metrics/collector.go

```diff
@@ -61,7 +61,7 @@ type StatMessage struct {
 type Collector interface {
 	// CreateOrUpdate either creates a collection for the given metric or update it, should
 	// it already exist.
-	CreateOrUpdate(*autoscalingv1alpha1.Metric) error
+	CreateOrUpdate(metric *autoscalingv1alpha1.Metric, usePassthroughLb bool) error
```

Comment:
I think this is property if the collector (given this flag is a state across all revisions). You should not need to pass it but rather set it at construction time (and cm update — even though this feels like a big enough flag to warrant restart).

Reply:
That's kind of what I had in mind, but hooking up the actual triggering seemed more trouble than it was worth. I can take another look tomorrow though.